sdk/core/azure-core/test/ut/curl_options_test.cpp

```diff
@@ -267,7 +267,7 @@ namespace Azure { namespace Core { namespace Test {
 
     std::unique_ptr<Azure::Core::Http::RawResponse> response;
     EXPECT_NO_THROW(response = pipeline.Send(request, Azure::Core::Context::ApplicationContext));
-    EXPECT_EQ(response->GetStatusCode(), Azure::Core::Http::HttpStatusCode::Ok);
+    EXPECT_NE(response->GetStatusCode(), Azure::Core::Http::HttpStatusCode::Ok);
```

Comment:
Intentional change to make sure our CI is running this test and failing, will revert.